coverage/cmdline.py

```
@@ -882,9 +884,10 @@ def do_debug(self, args: list[str]) -> int:
             print(info_header("data"))
             data_file = self.coverage.config.data_file
             debug_data_file(data_file)
-            for filename in combinable_files(data_file):
-                print("-----")
-                debug_data_file(filename)
+            with suppress(NoDataFilesFoundError):
```

Why do we need to suppress this error here?

---

We do not, but I tried to limit how many outputs I changed. Would you like this to display a friendly error instead?